robot/basestation/templates/elements/feed/RoverMotorTable.html

```diff
@@ -1,8 +1,8 @@
-<h5 class="container-header">Motor Table</h5>
+<h5 class="container-header">Wheel Motor Table</h5>
 <!--Table-->
 <table id="motor-table" class="table table-hover table-sm table-bordered" style="font-size:12px; background:rgba(255, 255, 255, 0.8); text-align: left">
```

Comment:
we should make the CSS in `style` a class, maybe `motor-table` or `rover-motor-table` if has distinctions from the arm motor table CSS?

Reply:
I think "Wheel Motor Table" instead of "Motor Table" is a good title since there's also the Arm Motor Table on the PDS page. I've also moved the CSS styling under `motor-table`.

Reply:
Fixed